src/Fixer/Operator/TernaryToNullCoalescingFixer.php

```diff
@@ -110,9 +60,65 @@ public function isCandidate(Tokens $tokens)
     }
 
     /**
+     * @param Tokens $tokens
+     * @param int    $index  of `T_ISSET` token
+     */
+    private function fixIsset(Tokens $tokens, $index)
+    {
+        $prevTokenIndex = $tokens->getPrevMeaningfulToken($index);
+        if ($this->isHigherPrecedenceAssociativityOperator($tokens[$prevTokenIndex])) {
+            return;
+        }
+
+        $startBraceIndex = $tokens->getNextTokenOfKind($index, array('('));
+        $endBraceIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $startBraceIndex);
+
+        $ternaryQuestionMarkIndex = $tokens->getNextMeaningfulToken($endBraceIndex);
+        if (!$tokens[$ternaryQuestionMarkIndex]->equals('?')) {
+            return; // we are not in a ternary operator
+        }
+
+        // search what is inside the isset()
+        $issetTokens = $this->getMeaningfulSequence($tokens, $startBraceIndex, $endBraceIndex);
+        if ($this->hasChangingContent($issetTokens)) {
+            return; // some weird stuff inside the isset
+        }
+
+        // search what is inside the middle argument of ternary operator
+        $ternaryColonIndex = $tokens->getNextTokenOfKind($ternaryQuestionMarkIndex, array(':'));
+        $ternaryFirstOperandTokens = $this->getMeaningfulSequence($tokens, $ternaryQuestionMarkIndex, $ternaryColonIndex);
+
+        if ($issetTokens->generateCode() !== $ternaryFirstOperandTokens->generateCode()) {
+            return; // regardless of non-meaningful tokens, the operands are different
+        }
+
+        $ternaryFirstOperandIndex = $tokens->getNextMeaningfulToken($ternaryQuestionMarkIndex);
+
+        // preserve comments and spaces
+        $comments = array();
+        $commentStarted = false;
+        $issetIndex = $index;
+        for (; $index < $ternaryFirstOperandIndex; ++$index) {
```

Comment:
This is weird: at half of the method the main variable name is changed to another one because you want that name inside a loop.
`for ($loopIndex = $index;` to me is enough for a meaningful variable name, and you preserve the main variable name all through the method.
Change this, squash and I'll merge 😉